PSix/CurrentUser.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CurrentUser.swift
+//  PSix
+//
+//  Created by Turzion, Avihu on 4/14/15.
+//  Copyright (c) 2015 PayPal. All rights reserved.
+//
+
+import Foundation
+import Parse
+
+class CurrentUser {
+    
+    private init() {}
```

Comment:
The idea was to have it be a non instantiable class so that only logging in would have the ability to create users. Or at least have as a utility class with all methods `static`. I'm not entirely sure how this class will be used when it comes to other users apart from the currently logged in user, so I did away with the `private init` at the moment, and will refactor it accordingly when the need arises.